### PR TITLE
fix #85. Add first iteration of Conclusion

### LINKED ISSUE #85: Conclusion

### PR DESCRIPTION
**Pull request by SebastianMygind**
I am not home tomorrow. So if you have anything you want to change please feel free to do so, as i think we need to have a conclusion when we send it to Andres.
You can also just merge it when it is ready as i am not home before late Sunday.

### CODE REVIEW

**Review by audio-engineer — DISMISSED** [review]

**Review by audio-engineer — DISMISSED** [review]

File: src/conclusion/conclusion.tex [hunk 1]
```diff
@@ -2,3 +2,59 @@
 \chapter{Conclusion}\label{ch:conclusion}
 % textidote: ignore end
 
+The game of chess is one that has stood the tests of time, this is partly due to the highly intricate mechanics that
+make chess an unsolved game.
+As chess is a game where all that matters is winning, much effort has through time has gone into experimenting with
+strategies and new inventions.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: Sentence reads weirdly

File: src/conclusion/conclusion.tex [hunk 2]
```diff
@@ -2,3 +2,59 @@
 \chapter{Conclusion}\label{ch:conclusion}
 % textidote: ignore end
 
+The game of chess is one that has stood the tests of time, this is partly due to the highly intricate mechanics that
+make chess an unsolved game.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: "test" in singular. New sentence after "time".

File: src/conclusion/conclusion.tex [hunk 3]
```diff
@@ -2,3 +2,59 @@
...
+As chess is a game where all that matters is winning, much effort has through time has gone into experimenting with
+strategies and new inventions.
+The latest step in chess has been computer chess engines which reached beyond human skill with Deep Blue.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: Replace "reached" with "reaches"

File: src/conclusion/conclusion.tex [hunk 4]
```diff
@@ -2,3 +2,59 @@
...
+Engines are now widely used for analysis, but the authors of this paper, have found that they also have immense
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: Second comma makes the senctence read weirdly. Consider removing it.

File: src/conclusion/conclusion.tex [hunk 5]
```diff
@@ -2,3 +2,59 @@
...
+potential for teaching chess.
+
+The saying ``Once a student, now the master'' shows that learning from masters of their craft is nothing new, but
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: I dont think this has been mentioned in the report, why we either have to delete it from the conclusion or include it somewhere in the report.

File: src/conclusion/conclusion.tex [hunk 6]
```diff
@@ -2,3 +2,59 @@
...
+teaching in chess usually comes in the form of a coach.
+Even if this approach is one of the most effective teaching tools it has its drawbacks.
+Coaching costs money and needs to be planned to a specific time.
+This excludes people from pursuing this teaching method for other cheaper or more accessible solutions, and this is
+where engines can be the answer.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: This sentence is good it just reads a little weird IMO.

File: src/conclusion/conclusion.tex [hunk 7]
```diff
@@ -2,3 +2,59 @@
...
+
+State-of-the-art engine implementation in web apps do not allow online multiplayer integration, which disallows the
+social aspects of chess while learning with engines.
+In this project the authors have come to the conclusion that this interaction between two chess players is very
+important both for the learning experience, but also the enjoyment in learning.
+To further improve the accessibility making the solution as a web app would also allow people to play any time of day
+one could want
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: Missing a period and read a bit weird IMO

File: src/conclusion/conclusion.tex [hunk 8]
```diff
@@ -2,3 +2,59 @@
...
+
+In terms of target audience simply because of how chess is written and talked about, the authors' solution,
+ChessTeacher, does not cater to the completely novice players.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: The start of this sentence makes no sense.

File: src/conclusion/conclusion.tex [hunk 9]
```diff
@@ -2,3 +2,59 @@
...
+These are players who do not know how or when specific moves can be made.
+The audience ChessTeacher will cater most to are beginners to intermediates, as they understand the rules and have begun
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: change "will cater" to "caters"

File: src/conclusion/conclusion.tex [hunk 10]
```diff
@@ -2,3 +2,59 @@
...
+learning patterns, which is also one aspect chess engines are superior to humans.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: I dont think this belongs in the conclusion. It does not really make sense to conclude on who the users is IMO

File: src/conclusion/conclusion.tex [hunk 11]
```diff
@@ -2,3 +2,59 @@
...
+
+While finishing the prototype user tests were made to understand how feasible the idea behind the project was, and also
+how good the implementation is.
+Although the reception of ChessTeacher was mixed, most of the critical feedback was related to the project only being
+a prototype and not what an eventual finished ChessTeacher would look like.
+This makes it certain that the approach has enough merit to be expanded and evolved further in future iterations.
+
+
+An interesting future for this approach to teaching chess would be testing how far up the ranking this approach is
+valid, as engines are the best chess players they should in theory also be the best teachers for chess, but this might
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: Maybe specify what ranking we are talking about.

File: src/conclusion/conclusion.tex [hunk 12]
```diff
@@ -2,3 +2,59 @@
...
+not be practical.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: I dont think this point has been discussed yet, why we cant conclude on it. Maybe add an issue for it or write about it in the discussion section

File: src/conclusion/conclusion.tex [hunk 13]
```diff
@@ -2,3 +2,59 @@
...
+
+The solution for this project is not perfect though and one important aspect of coaching has been overlooked in
+ChessTeacher.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: I dont think we should say overlooked, as we are well aware of the short commings. Maybe change the wording

File: src/conclusion/conclusion.tex [hunk 14]
```diff
@@ -2,3 +2,59 @@
...
+This is the way the feedback is presented to the players, as currently you are only presented with the moves and no
+context to why this move is good.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: I dont think we should be speaking directly to the reader. Consider changing the "you" to "users"

File: src/conclusion/conclusion.tex [hunk 15]
```diff
@@ -2,3 +2,59 @@
...
+The main limiting factor for this, is that chess engines are not made to communicate context of moves, but only find
+good moves.
+This means it cannot explain why it chose the move, and this step of the board analysis is just as critical as the
+actual move the engines suggest.
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: Maybe change to "... just as critical for the learning of the users as the ..." to make sure focus is on learning

File: src/conclusion/conclusion.tex [hunk 16]
```diff
@@ -2,3 +2,59 @@
...
+To solve this we need a system that outputs human-readable text in context to some input, and one rapidly evolving
+technology seems perfect for this part, and they are Large-Language-Models (LLMs).
```
Comment (MadsHeilmann) [inline src/conclusion/conclusion.tex]: No need for suspense in the sentence. Consider removing the suspence

**Review by MadsHeilmann — CHANGES_REQUESTED** [review]

**Comment by SebastianMygind** [comment]
@MadsHeilmann 
Valid feedback, but i do not have the time to change it before we need to send it to Andres, you can go onto my branch and do it, but i can't before we need to send it.

**Comment by MadsHeilmann** [comment]
> @MadsHeilmann Valid feedback, but i do not have the time to change it before we need to send it to Andres, you can go onto my branch and do it, but i can't before we need to send it.

All good. Will do :)

**Review by MadsHeilmann — DISMISSED** [review]

**Review by Danar435 — DISMISSED** [review]

**Review by Danar435 — APPROVED** [review]

**Comment by SebastianMygind** [comment]
This commit simply cleans the history and fixes merge problems. Still keeping @MadsHeilmann's changes

**Review by MadsHeilmann — APPROVED** [review]

**Pull request merged**